core/dbt/clients/jinja.py

```diff
@@ -632,3 +647,17 @@ def _convert_function(value: Any, keypath: Tuple[Union[str, int], ...]) -> Any:
     # when the test node was created in _parse_generic_test.
     kwargs = deep_map_render(_convert_function, node.test_metadata.kwargs)
     context[GENERIC_TEST_KWARGS_NAME] = kwargs
+
+
+def get_supported_languages(node: jinja2.nodes.Macro) -> List[ModelLanguage]:
+    no_args = not node.args or not node.defaults
```

Comment:
I am assuming the length of args is always gonna be longer than defaults(contain both args and kwargs)? So node.defaults doesn't need to be checked here?